dcc-portal-ui/app/scripts/survivalanalysis/views/survival-analysis.html

```diff
@@ -0,0 +1,31 @@
+<div class="survival-analysis">
+
+  <script type="text/template" class="survival-tip-template">
+    <div>
+      <strong><%= donor.id %></strong>
```

Comment:
lodash template, I couldn't figure out how to get this working with angular templates, kept trying to compile `{{}}`s